fir.lic

```diff
@@ -120,7 +120,7 @@ class FirQuest
       walk_to 19_073
       bput('steal silver vial in catalog', 'Roundtime')
     else
-      unless ensure_copper_on_hand(500, 'Crossing')
+      unless ensure_copper_on_hand(500, @settings)
```

Comment:
I'm not actually sure this will work if your hometown is one that uses Dok or Lir. You'd withdraw the wrong currency and then try to buy it with Kronars. You might want to check with the script author to be sure.

Reply:
Since we are not passing hometown or currency type anymore, not sure how to work around this at this moment. maybe state hometown must equal Crossing for this script to function?

Reply:
That seems reasonable to me. @skyleronken as the original author, what do you think?

Reply:
thats fine or maybe:

temp_settings = @settings
temp_settings.hometown = "Crossing"
unless ensure_copper_on_hand(500, temp_settings)

Reply:
Hello, 

Cannot send "Crossing," which is the same as trying to send @settings.hometown, as it will come back with a string error. In common-money it is settings.hometown, where settings is the whole of yaml settings.

Reply:
Right. That’s not what my code snippet is doing. It makes a copy of the settings object, then overwrites the hometown value of the copy to be “Crossing”. It then sends the modified settings. Essentially we manually override their hometown settings for that specific call.

Reply:
@Dartellum Did you try this change?

Reply:
I did not yet. I know from trying to pass in stuff on other attempts it errored. I can try though.